cmd/repository/main.go

```diff
@@ -199,6 +206,49 @@ func runServer(c *cli.Context) error {
 		}
 	}
 
+	// Spec format name@version:URL
+	for _, spec := range ensureSchemas {
+		reference, rawURL, ok := strings.Cut(spec, ":")
+		if !ok {
+			return errors.New("expected a specification in the format name@version:URL")
+		}
+
+		name, version, ok := strings.Cut(reference, "@")
+		if !ok {
+			return errors.New("expected a specification in the format name@version:URL")
+		}
+
+		uri, err := url.Parse(rawURL)
+		if err != nil {
+			return fmt.Errorf("invalid URL for the schema %s: %w", name, err)
+		}
+
+		var schema revisor.ConstraintSet
+
+		switch uri.Scheme {
+		case "file":
+			err := elephantine.UnmarshalFile(uri.Opaque, &schema)
+			if err != nil {
+				return fmt.Errorf("failed to load the schema file for %s: %w",
+					name, err)
+			}
+		case "http", "https":
+			err := elephantine.UnmarshalHTTPResource(rawURL, &schema)
+			if err != nil {
+				return fmt.Errorf("failed to load the schema %s over HTTP: %w",
```

Comment:
Very minor, but it might be worth saying "HTTP(S)" here in order not to confuse.